packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -20,14 +26,16 @@ export const NotOwnedSandboxInfo = () => {
       >
         <Files setEditActions={setEditActions} />
       </WorkspaceItem>
-
       <WorkspaceItem
         defaultOpen
         style={{ marginTop: '.5rem' }}
         title="Dependencies"
       >
         <Dependencies />
       </WorkspaceItem>
+      {(currentSandbox.git || {}).username === user.username && (
```

Comment:
The editor is broken when not signed in, because `user` can be `undefined` here.
```suggestion
      {user && (currentSandbox.git || {}).username === user.username && (
```